examples/supplychain-app/quorum/web3/controllers/product.js

```diff
@@ -0,0 +1,182 @@
+var Web3 = require("web3");
+var express = require("express"),
+  router = express();
+var bodyParser = require("body-parser");
+
+web3 = new Web3(new Web3.providers.HttpProvider("http://localhost:8545"));
```

Comment:
the IPaddress/ http address should not be hardcoded. use an environment variable.